test/unit/deferred.js

```diff
@@ -580,14 +582,17 @@ QUnit[ window.console ? "test" : "skip" ]( "jQuery.Deferred.exceptionHook", func
 			// Should NOT get an error
 			throw new Error( "Make me a sandwich" );
 		} ).then( null, jQuery.noop )
-	).then( function( ) {
+	).then( function barf( ) {
+		jQuery.thisDiesToo();
+	} ).then( null, function( ) {
 		window.console.warn = oldWarn;
 		done();
 	} );
 
 	defer.resolve();
 } );
 
+// Test fails in IE9 but is skipped there because console is not active
```

Comment:
Oh wait, I just remembered:
>Different browsers set this value at different times. For example, Firefox sets it when creating an Error object, while PhantomJS sets it only when throwing the Error, and MSDN docs also seem to match the PhantomJS implementation. -- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Stack

So the test would need to be more elaborate. I think the current check is less likely to skip by accident at least? Everyone but IE9 has a console.